config/config.go

```diff
@@ -140,10 +156,66 @@ func (c *Config) validateIntegrationTokens() error {
 }
 
 func (c *Config) validateAliases() error {
-	for name := range c.Aliases {
+	validCommands := map[string]bool{
```

Comment:
Isn't the validCommand function actually unnecessary? 👀 

The existence of the command is already checked on the router.go side, so it seemed unnecessary here.